src/Ldap.php

```diff
@@ -92,7 +111,7 @@ public function __construct($options = [])
      */
     public function __destruct()
     {
-        $this->disconnect();
+        $this->unbind();
```

Comment:
I understand that it does the same. But from a naming POV I'd definitely prefer the ```disconnect``` here.

Reply:
I couldn't find any justification for why I changed this. Will remove the change.